docs/modules/ROOT/pages/index.adoc

```diff
@@ -24,18 +24,23 @@ Users of Camel K can instantly run integration code written in Camel DSL on thei
 [[how-it-works]]
 == How It Works
 
-Just write a _helloworld.groovy_ integration file with the following content:
-
-```groovy
-from('timer:tick?period=3000')
-  .setBody().constant('Hello world from Camel K')
-  .to('log:info')
+Just write a _helloworld.yaml_ integration file with the following content:
+
+```yaml
+- from:
+    uri: "timer:yaml"
```

Comment:
I'd leave the `timer:tick` instead. Any newcomer may confuse the name of the component with the DSL.